src/main/riddl/Venues/Reservations/reservationTypeDefinitions.riddl

```diff
@@ -0,0 +1,23 @@
+type Status is any of {Draft, OnHold, Reserved, Active, Past} briefly "Internal status of a Reservation."
+
+type Info is {
```

Comment:
I wonder if reservation also need to have a reference to an Event? A reservation is made by an organization, during TimeRange, at Location, for Event